src/functions/notify/function_app.py

```diff
@@ -0,0 +1,25 @@
+import json
+import logging
+import sys
+import os
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
+
+from helper import send_messages
```

Comment:
https://github.com/NHSDigital/dtos-communication-management/pull/11/commits/924f9cc52869c9428dd325308db6576e106e4980 addresses this, I can see pros and cons. I agree it's useful to know where a function is imported from, and this works well when we are just importing functions rather than classes and calling methods on them.